src/client/api.ts

```diff
@@ -219,6 +220,7 @@ export const depositInvestment = multiAsync(
 			connection,
 			wallet
 		);
+		const _getCredixPass = findCredixPassPDA(wallet.publicKey);
```

Comment:
maybe also `_getCredixPassPDA` to make it clear that it's a pda, same with `credixPassPDA` below